dao/src/test/java/org/thingsboard/server/dao/sql/user/JpaUserDaoTest.java

```diff
@@ -45,35 +46,54 @@
  */
 public class JpaUserDaoTest extends AbstractJpaDaoTest {
 
+    // add to count user sysadmin@thingsboard.org
```

Comment:
Put the explanation where is that user appears from. Like  "it comes from the ____ super class"